index.html

```diff
@@ -65,10 +65,20 @@
       <!-- Example row of columns -->
       <div class="row">
         <div class="col-md-4">
-          <h2>Heading</h2>
-          <p>Donec id elit non mi porta gravida at eget metus. Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus. Etiam porta sem malesuada magna mollis euismod. Donec sed odio dui. </p>
-          <p><a class="btn btn-default" href="#" role="button">View details &raquo;</a></p>
+          <h2>Friends</h2>
+            <p>(35)</p>
+        <div class="friends">
+            <div class="each-friend friend-1"> Jane Doe</div>
+                <img src="images/blondelady.jpeg">
+            <div class="each-friend friend-2"> Josi Doe</div>
+                <img src="images/brunettelady.jpg">
+            <div class="each-friend friend-3"> Jim Doe</div>
+                <img src="images/brunetteman.jpeg">
```

Comment:
Maybe there could be a button added after each picture to view each friends profile.